tests/integration/common/auth/test_auth.py

```diff
@@ -192,6 +193,42 @@ def test_get_without_fsd_admin_role(self, app, anonymous_client):
         assert response.status_code == 403
         assert "https://mhclgdigital.atlassian.net/servicedesk/customer/portal/5" in response.text
 
+    def test_get_without_fsd_admin_role_and_with_invite(self, app, anonymous_client, factories):
+        with patch("app.common.auth.build_msal_app") as mock_build_msap_app:
+            user = factories.user.create(email="test.member@communities.gov.uk")
+            grant = factories.grant.create()
+            factories.user_role.create(user_id=user.id, user=user, role=RoleEnum.MEMBER, grant=grant)
+            # Partially mock the expected return value; just enough for the test.
+            mock_build_msap_app.return_value.acquire_token_by_auth_code_flow.return_value = {
+                "id_token_claims": {
+                    "preferred_username": "test.member@communities.gov.uk",
+                    "name": "SSO User",
+                    "roles": [],
+                }
+            }
+
+            response = anonymous_client.get(url_for("auth.sso_get_token"), follow_redirects=True)
+
+        assert response.status_code == 200
+        soup = BeautifulSoup(response.data, "html.parser")
+        nav_items = [li.get_text(strip=True) for li in soup.find("ul", id="navigation").find_all("li")]
+        assert len({"Home", "Grant details", "Grant team"} - set(nav_items)) == 0
```

Comment:
I don't think it's good to assert on the nav bar here - changes to that will break this test and it's slightly unrelated.

Maybe we just check that the endpoint sends a redirect to the expected place?